apiary.apib

```diff
@@ -374,6 +374,10 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- The task assignee is now nullable: you can create / update unassigned tasks by sending null as assignee. The tasks.list and tasks.info endpoints also return null as assignee in case the task is unassigned.
```

Comment:
I imagine you can also not send the assignee?

Reply:
Yes, although that automagically assigns it to yourself, it seems :D. We'll need to fix that bug.

Reply:
But the assignee was previously documented as optional (both in create and in update), so it's not something to add here.